launch/app/app.bndrun

```diff
@@ -231,65 +231,66 @@ feature.openhab-model-runtime-all: \
 	org.osgi.util.function;version='[1.2.0,1.2.1)',\
 	org.osgi.util.promise;version='[1.3.0,1.3.1)',\
 	org.threeten.extra;version='[1.5.0,1.5.1)',\
-	org.yaml.snakeyaml;version='[2.0.0,2.0.1)',\
+	org.yaml.snakeyaml;version='[2.2.0,2.2.1)',\
```

Comment:
just noticed that this was not in sync before... in core it was 2.1 before I upgraded to 2.2

Reply:
I think I upgraded it to 2.1 after 2.15.2 got released and before it was added as dependency in 2.15.3.

It didn't seem that risky:

https://bitbucket.org/snakeyaml/snakeyaml/wiki/Changes